tests/utils/test_db.py

```diff
@@ -32,10 +33,12 @@
 from alembic.script import ScriptDirectory
 from sqlalchemy import MetaData
 
+import airflow
```

Comment:
Should we make this import at the point of use?

Reply:
Yes we can do that, makes sense too.